### PR TITLE
[BUG] Fixed a bug and added a feature simultaneously!

### PR DESCRIPTION
**Pull request by 0x48piraj**
## Fixes https://github.com/fossasia/gci17.fossasia.org/issues/409

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [ ] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request
- Fixed a bug and added a feature simultaneously! :heart: 
- Added a randomizer to shuffle the projects and show new projects each time a user visits the page.
- **_Preview : https://0x48piraj.github.io/gci17.fossasia.org/repos.html_**

### CODE REVIEW

**Review by anshumanv — APPROVED** [review]

**Review by anshumanv — APPROVED** [review]
LGTM

File: js/contributors.js [hunk 1]
```diff
@@ -1,3 +1,9 @@
+/* Used to get randomize repositories by using the Github API */
+function Shuffle(o) {
```
Comment (vaibhavsingh97) [inline js/contributors.js]: use `shuffle` instead of `Shuffle`

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]
Reference: 
1. https://www.drupal.org/docs/develop/standards/javascript/javascript-coding-standards#functions
2. https://stackoverflow.com/questions/1564398/javascript-method-naming-lowercase-vs-uppercase

**Review by m1guelpf — APPROVED** [review]

**Comment by 0x48piraj** [comment]
Thanks a lot @vaibhavsingh97 :sparkling_heart: 
Squashed the commits too.

**Comment by vaibhavsingh97** [comment]
Please change commit message as `Fixes #<issue-number>: <commit message>`

**Review by vaibhavsingh97 — APPROVED** [review]
Good Work!! LGTM

**Comment by 0x48piraj** [comment]
Thanks @vaibhavsingh97 @anshumanv @m1guelpf @harshit98 for your valuable feedbacks. :sparkles:

**Comment by 0x48piraj** [comment]
Thanks a lot @harshit98 :100:

**Pull request merged**